Peli-Kiera/README.md

```diff
@@ -78,17 +101,18 @@ from __future__ import unicode_literals
 AUTHOR = 'Author'
 COPYRIGHT = '2019'
 SITENAME = 'Site-Name'
-SITEURL = ''
-SITESUBTITLE = Site Subtitle'
+SITEURL = 'https://my-pelican-blog-address.io'
```

Comment:
Might I suggest using https://www.example.com here instead? Browsers will at least render that properly, and it avoids potential situations where malware could be (eventually) hosted on this currently-fictitious domain. 😊

Reply:
Great call.. Thank you!